distributed/utils.py

```diff
@@ -359,12 +359,17 @@ def convert(task):
 
 import logging
 from .compatibility import logging_names
-logging.basicConfig(format='%(name)s - %(levelname)s - %(message)s',
-                    level=logging.INFO)
+
+handler = logging.StreamHandler(sys.stderr)
+handler.setLevel(logging.INFO)
+fmt = logging.Formatter('%(name)s - %(levelname)s - %(message)s')
+handler.setFormatter(fmt)
 
 for name, level in config.get('logging', {}).items():
```

Comment:
Right, you probably wouldn't ever want all output to be silenced. I'm assuming that the test is failing because no loggers are configured so I've pushed a further change to fall back to `basicConfig` in that instance. We'll see what travis thinks of that...

I've just been using the GitHub web-interface so its easier for me to simply wait for travis. I can squash the commits if desired when the test pass.